WordPress/src/main/java/org/wordpress/android/ui/stats/StatsFollowersFragment.java

```diff
@@ -288,7 +288,7 @@ public void onClick(View view) {
             holder.totalsTextView.setText(getSinceLabel(currentRowData.getDateSubscribed()));
 
             // Avatar
-            holder.networkImageView.setImageUrl(PhotonUtils.fixAvatar(currentRowData.getAvatar(), mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.AVATAR);
+            holder.networkImageView.setImageUrl(GravatarUtils.fixGravatarUrl(currentRowData.getAvatar(), mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.AVATAR);
```

Comment:
OK, I'll change this. But keep in mind that WPNetworkImageView makes the downloaded image circular [here](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2311-reader-blavatar/WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java#L250) when the image type is an avatar.